resources/config/varnish/fos_purge_xkey.vcl

```diff
@@ -0,0 +1,29 @@
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import xkey;
+
+sub fos_purge_recv {
+    if (req.method == "PURGE") {
```

Comment:
i would prefer to use a specific method for key purge. that makes it more modular